ruby.md

```diff
@@ -5,41 +5,46 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. In what ways are JavaScript and Ruby similar? In what ways are they different?
 
-  Your answer:
+  Your answer:They both use object oriented computer languages, they have built in methods.
 
-  Researched answer:
+  Researched answer:JavaScript is an object-oriented scripting language that allows you to create dynamic HTML pages with interactive effects within a webpage. JavaScript allows you to process and maintain data within the browser. It runs on web-browser only and it is an interpreted programming language. Ruby is an open-source programming language, object-oriented language with a focus on simplicity, portability, extensibility, and productivity. As it is open-source, we can free to modify, edit according to our needs and use.
+  
+  In javascript you use semicolon's in ruby it's not an option. In ruby you use "end" and in java script you use open and closing curly brackets.
+  in Ruby you dont need parenthesis for methods but in java they are required
 
```

Comment:
Although I think this was just a type-o, don't forget that Java and JavaScript are two very different things. Ruby is also an interpreted programming language.